hack/build.sh

```diff
@@ -129,6 +129,11 @@ source_lint() {
 go_build() {
   echo "🚧 Compile"
   go build -mod=vendor -ldflags "$(build_flags $(basedir))" -o kn ./cmd/...
+  WIN="Windows"
+  FILE=$(file ./kn)
+  if grep -q "$WIN" <<< "$FILE"; then
+    mv ./kn ./kn.exe
+  fi
```

Comment:
```suggestion
  if $(file kn | grep -q -i "Windows"); then
    mv kn kn.exe
  fi
```

This is a more canonical way for this checking (and also does not pollute with global variables.

Reply:
Done